src/classes/os-utils.ts

```diff
@@ -53,40 +53,19 @@ export default class OsUtils {
     }
   }
 
-    /**
+  /**
    * Re-writes user config using previous user config as basis
    * 
    * @param os is the OS that is being switched to, if supported
    */
   private static async swapOs (os : string) {
-    const userConfig : UserConfigJson = await OsUtils.readUserConfig();
+    const userConfig : UserConfigJson = await userConfigUtils.readUserConfig();
     userConfig.osPref = os;
     const data = JSON.stringify(userConfig, null, 2);
-
-    try {
-      fs.writeFile("./user-config.json", data, "utf8", (err) => {
-        if (err) {
-          throw err;
-        }
-      });
-    } catch (err) {
-      throw new Error("Error when attempting to write file:" + err);
-    }
+    await userConfigUtils.writeUserConfig(data);
```

Comment:
Class names should be captitalized
```suggestion
    await UserConfigUtils.writeUserConfig(data);
```